src/plugins/vis_type_vega/public/default.spec.hjson

```diff
@@ -1,31 +1,31 @@
 {
-/*
+  /*
```

Comment:
Reviewers - sorry for sneaking in a somewhat unrelated change, but the lack of indentation in the tutorial spec has been driving me nuts for weeks, and I couldn't resist fixing while I was already editing this file.